src/attributes/components/AttributePage/AttributePage.tsx

```diff
@@ -172,7 +176,10 @@ const AttributePage: React.FC<AttributePageProps> = ({
                   errors={errors}
                   onChange={change}
                 />
-                {data.inputType !== AttributeInputTypeEnum.FILE && (
+                {![
+                  AttributeInputTypeEnum.FILE,
```

Comment:
please move this array to a named const, so it's obvious what type of inputs those are (like here I think `attributesInputTypesWithValues` or something like that?)